### DISCUSSION TITLE
How use Redis cache in loopback 4?

### DISCUSSION THREAD

**Discussion by dgamepuzzle** [loopbackio/loopback-next#7150] [Q&A]
How i can find example use redis cache in loopback 4 interceptor ?

**Comment by marioestradarosa** [comment]
Hi @dgamepuzzle , what is your use case scenario ?  Have you looked at the [Shopping cart example](https://github.com/strongloop/loopback4-example-shopping) , it implements redis and uses the key value repository for it.
